UnitTests/UnitTests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>UnitTests Class Library</Description>
     <Authors>Administrator</Authors>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
```

Comment:
m? any reason to return 2.1 in test project?

Reply:
Yes - again see https://github.com/streeva/jose-jwt/runs/1645966664?check_suite_focus=true.
The action is run as a matrix so you can see what is failing or passing running against both netcoreapp2.1 and netcoreapp3.1. Also the difference between windows / linux os.
Currently only windows netcoreapp3.1 of those combinations has a clean run - but that is basically the existing reality (I believe).